### ISSUE TITLE
[Bug] LUA error on login 10.2.7: bad argument #1 to 'SetJustifyV' (Usage: self:SetJustifyV(justifyV))

### ISSUE THREAD

**Issue by RussB-626** [fang2hou/ElvUI_WindTools#387]
**Describe the bug**
LUA error on login 10.2.7: bad argument #1 to 'SetJustifyV' (Usage: self:SetJustifyV(justifyV))

**To Reproduce**
- Have the current version of ElvUI WindTools version 3.55 installed for WoW Retail version 10.2.6 installed
- Log into the game and verify LUA error pops up

**Expected behavior**
Self explanatory

**Screenshots**
![image](https://github.com/fang2hou/ElvUI_WindTools/assets/37383310/a270725c-eca6-43ea-84a6-d797e0a20e7c)

**Information (you can find these in [WindTools-Information]):**
 - Addon Version: 3.55-fix released 5/7/2024
 - ElvUI Version: v13.64
 - WoW Build: retail 10.2.7

**Additional context**

```
Message: ...erface/AddOns/ElvUI_WindTools/Core/Compatibility.lua:105: bad argument #1 to 'SetJustifyV' (Usage: self:SetJustifyV(justifyV))
Time: Tue May  7 18:14:00 2024
Count: 1
Stack: ...erface/AddOns/ElvUI_WindTools/Core/Compatibility.lua:105: bad argument #1 to 'SetJustifyV' (Usage: self:SetJustifyV(justifyV))
[string "@Interface/AddOns/ElvUI_WindTools/Core/Compatibility.lua"]:268: in function `CheckCompatibility'
[string "@Interface/AddOns/ElvUI_WindTools/Initialize.lua"]:102: in function `?'
[string "@Interface/AddOns/BetterBags/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua"]:109: in function <...ags/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:109>
[string "=[C]"]: ?
[string "@Interface/AddOns/BetterBags/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua"]:19: in function <...ags/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:15>
[string "@Interface/AddOns/BetterBags/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua"]:54: in function `Fire'
[string "@Interface/AddOns/BetterBags/libs/AceEvent-3.0/AceEvent-3.0.lua"]:120: in function <...AddOns/BetterBags/libs/AceEvent-3.0/AceEvent-3.0.lua:119>

Locals: self = <table> {
 SetDefaultModuleLibraries = <function> defined @Interface/AddOns/BetterBags/libs/AceAddon-3.0/AceAddon-3.0.lua:363
 EnableModule = <function> defined @Interface/AddOns/BetterBags/libs/AceAddon-3.0/AceAddon-3.0.lua:328
 modules = <table> {
 }
 CancelTimer = <function> defined @Interface/AddOns/WeakAuras/Libs/AceTimer-3.0/AceTimer-3.0.lua:147
 IterateEmbeds = <function> defined @Interface/AddOns/BetterBags/libs/AceAddon-3.0/AceAddon-3.0.lua:438
 SecureHookScript = <function> defined @Interface/AddOns/BetterBags/libs/AceHook-3.0/AceHook-3.0.lua:409
 ConstructCompatibilityFrame = <function> defined @Interface/AddOns/ElvUI_WindTools/Core/Compatibility.lua:18
 RawHookScript = <function> defined @Interface/AddOns/BetterBags/libs/AceHook-3.0/AceHook-3.0.lua:394
 Printf = <function> defined @Interface/AddOns/BetterBags/libs/AceConsole-3.0/AceConsole-3.0.lua:65
 AddCommand = <function> defined @Interface/AddOns/ElvUI_WindTools/Core/Commands.lua:22
 SetDefaultModulePrototype = <function> defined @Interface/AddOns/BetterBags/libs/AceAddon-3.0/AceAddon-3.0.lua:405
 defaultModuleState = true
 IsEnabled = <function> defined @Interface/AddOns/BetterBags/libs/AceAddon-3.0/AceAddon-3.0.lua:447
 ScheduleTimer = <function> defined @Interface/AddOns/WeakAuras/Libs/AceTimer-3.0/AceTimer-3.0.lua:94
 DisableModule = <function> defined @Interface/AddOns/BetterBags/libs/AceAddon-3.0/AceAddon-3.0.lua:346
 RegisterModule = <function> defined @Interface/AddOns/ElvUI_WindTools/Core/Core.lua:93
 RegisteredModules = <table> {
 }
 RegisterMessage = <function> defined @Interface/AddOns/BetterBags/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:80
 UnregisterMessage = <function> defined @Interface/AddOns/BetterBags/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:143
 hooks = <table> {
 }
 CompatibleFont = false
 SetEnabledState = <function> defined @Interface/AddOns/BetterBags/libs/AceAddon-3.0/AceAddon-3.0.lua:420
 Hook = <function> defined @Interface/AddOns/BetterBags/libs/AceHook-3.0/AceHook-3.0.lua:273
 enabledState = true
 RegisterEvent = <function> defined @Interface/AddOns/BetterBags/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:80
 Utilities = <table> {
 }
 IterateModules = <function> defined @Interface/AddOns/BetterBags/libs/AceAddon-3.0/AceAddon-3.0.lua:433
 Version = "3.55"
 RawHook = <function> defined @Interface/AddOns/BetterBags/libs/AceHook-3.0/AceHook-3.0.lua:312
 Media = <table> {
 }
 Enable = <function> defined @Interface/AddOns/BetterBags/libs/AceAddon-3.0/AceAddon-3.0.lua:290
 UnregisterAllEvents = <function> defined @Interface/AddOns/BetterBags/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:164
 PlainTitle = "WindTools"
 Locale = "enUS"
 GetModule = <function> defined @Interface/AddOns/BetterBags/libs/AceAddon-3.0/AceAddon-3.0.lua:206
 defaultModuleLibraries = <table> {
 }
 TimeLeft = <function> defined @In
```

**Comment by Fraktyl** [comment]
Change the word "CENTER" to "MIDDLE" on line 105
`    completeButton.Text:SetJustifyV("MIDDLE")`

**Comment by fang2hou** [comment]
fixed in 1aabc22ed262b61842bb83ab4edbd896dafbd49f
